surveys/templates/partials/sidebar.html

```diff
@@ -46,6 +32,23 @@ <h3><b>Just Spaces</b></h3>
                     View collected data
                 </a>
             </li>
+            <li>
+                <a href="#createSurveySubmenu" data-toggle="collapse" aria-expanded="false" class="dropdown-toggle">
+                    <i class="fas fa-sm fa-home"></i>
```

Comment:
What do you think about `fa-gear` here? `fa-home` doesn't feel quite right, but it's hard to think of a good alternative for `Admin`.

Reply:
Love it

Reply:
Update: It's `fa-cog`